modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -499,6 +448,167 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
+          [Controls.OPAIntegration]: clusterSpec?.opaIntegration?.enabled,
+          [Controls.Konnectivity]: clusterSpec?.clusterNetwork?.konnectivityEnabled,
+          [Controls.MLALogging]: clusterSpec?.mla?.loggingEnabled,
+          [Controls.KubernetesDashboardEnabled]: clusterSpec?.kubernetesDashboard?.enabled,
+          [Controls.MLAMonitoring]: clusterSpec?.mla?.monitoringEnabled,
+          [Controls.AdmissionPlugins]: clusterSpec?.admissionPlugins,
+          [Controls.EventRateLimitConfig]: clusterSpec?.eventRateLimitConfig,
+          [Controls.PodNodeSelectorAdmissionPluginConfig]: clusterSpec?.podNodeSelectorAdmissionPluginConfig,
+          [Controls.ProxyMode]: clusterSpec?.clusterNetwork?.proxyMode,
+          [Controls.CNIPluginVersion]: clusterSpec?.cniPlugin?.version ?? '',
+          [Controls.IPv4CIDRMaskSize]: clusterSpec?.clusterNetwork?.nodeCidrMaskSizeIPv4,
+          [Controls.IPv6CIDRMaskSize]: clusterSpec?.clusterNetwork?.nodeCidrMaskSizeIPv6,
+          [Controls.NodeLocalDNSCache]: clusterSpec?.clusterNetwork?.nodeLocalDNSCacheEnabled,
+          [Controls.ExposeStrategy]: clusterSpec?.exposeStrategy,
+          [Controls.APIServerAllowedIPRanges]: clusterSpec?.apiServerAllowedIPRanges?.cidrBlocks,
+          [Controls.IPv4PodsCIDR]: NetworkRanges.ipv4CIDR(clusterSpec?.clusterNetwork?.pods),
+          [Controls.IPv6PodsCIDR]: NetworkRanges.ipv6CIDR(clusterSpec?.clusterNetwork?.pods),
+          [Controls.IPv4ServicesCIDR]: NetworkRanges.ipv4CIDR(clusterSpec?.clusterNetwork?.services),
+          [Controls.IPv6ServicesCIDR]: NetworkRanges.ipv6CIDR(clusterSpec?.clusterNetwork?.services),
+        });
+
+        // Selective patching of the form values to avoid trigger of valueChanges
+        // and to avoid resetting of the form values.
+        this.form.patchValue(
+          {
+            [Controls.IPFamily]: clusterSpec?.clusterNetwork?.ipFamily,
+            [Controls.CNIPlugin]: clusterSpec?.cniPlugin?.type,
+          },
+          {emitEvent: false}
+        );
+
+        if (!this.isDualStackAllowed) {
+          this.control(Controls.IPFamily).reset();
+          this.control(Controls.IPFamily).setValue(IPFamily.IPv4);
+        }
+
+        if (this.isAllowedIPRangeSupported()) {
+          this.form.patchValue({
+            [Controls.IPv4AllowedIPRange]:
+              NetworkRanges.ipv4CIDR(this._getExtraCloudSpecOptions().nodePortsAllowedIPRanges) ??
+              this._defaultAllowedIPRangeIPv4,
+            [Controls.IPv6AllowedIPRange]:
+              NetworkRanges.ipv6CIDR(this._getExtraCloudSpecOptions().nodePortsAllowedIPRanges) ??
+              this._defaultAllowedIPRangeIPv6,
+          });
+        } else {
+          this.form.patchValue({
+            [Controls.IPv4AllowedIPRange]: null,
+            [Controls.IPv6AllowedIPRange]: null,
+          });
+        }
+
+        // Based {emitEvent: false} the valueChanges will not be triggered.
```

Comment:
I think this comment doesn't add any value (at least I couldn't understand the context)